collections.go

```diff
@@ -31,6 +36,40 @@ func (cl *CollectionsClient) Create(name string) *CollectionsCreateClient {
 	return newCollectionsCreateClient(cl.sl, name)
 }
 
+type CollectionsStructureClient struct {
+	sl *sling.Sling
+}
+
+func newCollectionsStructureClient(sl *sling.Sling, id CollectionID) *CollectionsStructureClient {
+	data := struct {
+		ID CollectionID `json:"id"`
+	}{ID: id}
+
+	copy := sl.New()
+	copy.Post(common.CollectionsStructureEndpoint()).BodyJSON(&data)
+
+	return &CollectionsStructureClient{sl: copy}
+}
+
+type DocumentStructure []DocumentSummary
```

Comment:
Is this a typicall Go pattern? 🤔 
Why not simply returning a `[]DocumentSummary`? 🤔

Reply:
Well it is more case-by-case basis. Here we do it to be more in line with other API methods that is returning a single type representing the whole model and not a list/map of types.